<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HelloWindow</class>
 <widget class="QMainWindow" name="HelloWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>884</width>
    <height>826</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="main">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>800</width>
     <height>0</height>
    </size>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="styleSheet">
       <string notr="true">backgroud-color: rgb(246, 245, 244)</string>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="loginPage">
       <property name="sizeIncrement">
        <size>
         <width>10</width>
         <height>10</height>
        </size>
       </property>
       <widget class="QWidget" name="loginLayot" native="true">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>230</y>
          <width>411</width>
          <height>271</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(246, 245, 244)</string>
        </property>
        <layout class="QVBoxLayout" name="loginLayout">
         <item>
          <layout class="QHBoxLayout" name="loginInputLayot">
           <item>
            <widget class="QLabel" name="loginLoginLabel">
             <property name="styleSheet">
              <string notr="true">font: 25 12pt &quot;Ubuntu Light&quot;;</string>
             </property>
             <property name="text">
              <string>Логин   </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="login"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="passwordLoginInputLayout">
           <item>
            <widget class="QLabel" name="passwordLoginLabel">
             <property name="styleSheet">
              <string notr="true">font: 25 12pt &quot;Ubuntu Light&quot;;</string>
             </property>
             <property name="text">
              <string>Пароль</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="password"/>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="loginButton">
           <property name="text">
            <string>Вход</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="registrationButton">
           <property name="text">
            <string>Регистрация</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="registrationPage">
       <widget class="QWidget" name="registrationLayot" native="true">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>200</y>
          <width>461</width>
          <height>311</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(246, 245, 244)</string>
        </property>
        <layout class="QVBoxLayout" name="registrationLayout">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QGridLayout" name="registrationGridInput">
             <item row="0" column="0">
              <widget class="QLabel" name="loginLabelRegistration">
               <property name="styleSheet">
                <string notr="true">font: 25 12pt &quot;Ubuntu Light&quot;;</string>
               </property>
               <property name="text">
                <string>Логин   </string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="loginRegistration"/>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="passwordLabelRegistration">
               <property name="styleSheet">
                <string notr="true">font: 25 12pt &quot;Ubuntu Light&quot;;</string>
               </property>
               <property name="text">
                <string>Пароль</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="passwordRegistration"/>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QPushButton" name="registrationRegistrationButton">
             <property name="text">
              <string>Регистрация</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="backToLoginWindowButton">
             <property name="text">
              <string>Назад</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>884</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
